src/index.tsx

```diff
@@ -87,12 +87,12 @@ export const App = () => {
                                 <RemoteConfigContextProvider>
                                   <FirestoreConfigurationContextProvider>
                                     <TicketingContextProvider>
-                                      <MobileTokenContextProvider>
-                                        <AppLanguageProvider>
-                                          <GeolocationContextProvider>
-                                            <GlobalMessagesContextProvider>
-                                              <AnnouncementsContextProvider>
-                                                <ReactQueryProvider>
+                                      <ReactQueryProvider>
```

Comment:
Lets start with having it here. I'm not really sure what the best level is. It should at least be below the `AuthContext`, since some requests need to send the id token.